group.go

```diff
@@ -15,7 +15,7 @@ type Group struct {
 	GroupId      string                 `json:"groupId"`
 	Timestamp    time.Time              `json:"timestamp,omitempty"`
 	Context      *Context               `json:"context,omitempty"`
-	Traits       map[string]interface{} `json:"traits,omitempty"`
+	Traits       Traits                 `json:"traits,omitempty"`
```

Comment:
No, the Go type system doesn't allow it, `type Traits map[string]interface{}` is not assignable to `map[string]interface{}`.

Although this syntax is still supported:
```go
group := Group{
    ...
    Traits: analytics.Traits{
        "name": "foo",
    }
}
```
So it's just a matter of typing `analytics.Traits` instead of map[string]interface{}, and if the code was using `map[string]interface{}` already it can be easily converted as well:
```go
traits := map[string]interface{}{
    "name": "foo",
}

group := Group{
    ...
    Traits: analytics.Traits(traits),
}
```